wandb/sdk/wandb_run.py

```diff
@@ -3703,18 +3703,18 @@ def _header_wandb_version_info(self) -> None:
         self._printer.display(f"Tracking run with wandb version {wandb.__version__}")
 
     def _header_sync_info(self) -> None:
+        sync_dir = self._settings.sync_dir
+        info = [f"Run data is saved locally in {self._printer.files(sync_dir)}"]
+
         if self._settings._offline:
-            self._printer.display(
-                [
-                    f"W&B syncing is set to {self._printer.code('`offline`')}"
-                    f" in this directory. Run {self._printer.code('`wandb online`')}"
-                    f" or set {self._printer.code('WANDB_MODE=online')}"
-                    " to enable cloud syncing.",
-                ]
-            )
+            info = [
+                f"W&B syncing is set to {self._printer.code('`offline`')}"
+                f" in this directory. Run {self._printer.code('`wandb online`')}"
+                f" or set {self._printer.code('WANDB_MODE=online')}"
+                " to enable cloud syncing.",
+            ] + info
+            self._printer.display(info)
```

Comment:
I think it'd be neater as

```python
self._printer.display(offlineWarning + syncLocation)
```

instead of redefining `info`. Same in the other branch. Makes it immediately clear at the `display()` call what will be displayed.